detekt.gradle

```diff
@@ -1,19 +1,8 @@
-configurations {
-    detekt
-}
-
-dependencies {
-    detekt LibsDependencies.detekt
-}
+apply plugin: 'io.gitlab.arturbosch.detekt'
 
-task detekt(type: JavaExec) {
-    main = "io.gitlab.arturbosch.detekt.cli.Main"
-    classpath = configurations.detekt
-    def input = "$projectDir"
-    def config = "$rootDir/default-detekt-config.yml"
-    def exclude = ".*test.*,.*/resources/.*,.*/tmp/.*"
-    def params = ['-i', input, '-c', config, '-ex', exclude, '-r', '--fail-fast']
-    args(params)
+detekt {
+    config = files("$rootDir/default-detekt-config.yml")
+     //  filters = ".*build.*,.*/resources/.*,.*/tmp/.*"
+    //Optional baseline, uncomment & run gradle command detektBaseline to exclude existing issues
+    //  baseline = file("detekt-baseline.xml")
```

Comment:
if the task is a comment .. How can It work? 
I change the format, names and some code style and the task did not identify it. 
Please update it, and explain to us how can I see that the task is working.

Reply:
@aliceresponde the baseline is used to add "exclusions" for existing code that is hard to change at the moment, for example in a huge project where lint is implemented recently and in class X is hard to make the changes, the baseline will have reference to class X where the lint would fail to let the build pass (in continuous integration) without marking a lint error.